.github/workflows/pytest-benchmark.yml

```diff
@@ -3,6 +3,9 @@ on:
   push:
     branches:
       - master
+    paths:
+      - "syft/frameworks/torch/tensors/**"
+      - "syft/frameworks/torch/mpc/**"
```

Comment:
@aanurraj @gmuraru do these paths seem ok to you?

Reply:
Seems good to me! Let's see what George thinks...🤔